### ISSUE TITLE
Sub-optimal results for BarlowTwins in "Towards Democratizing Joint-Embedding Self-Supervised Learning"

### ISSUE THREAD

**Issue by gurdeepmaurya** [facebookresearch/FFCV-SSL#13]
Hi @bordesf ,

The best result in the respective publication, "Towards Democratizing Joint-Embedding Self-Supervised Learning," reports BarlowTwins accuracy as  66.8 with AdamW and ~68 with LARS optimizer; 

However, in the original publication of BarlowTwins, the accuracy for a batch size of 256 ~70.7 (see attached Image).

Could you please comment on the optimality of hyper-parameters, and how they are better than the original publication.

Thanks and Regards

![BT_batchsize_accuracy_variation](https://github.com/facebookresearch/FFCV-SSL/assets/42508824/c8a95b14-1152-4e89-9dee-f39a7a1fb284)

**Comment by bordesf** [comment]
Hi @gurdeepmaurya, so from what I remember from the Barlow Twins paper, the authors trained their models for 300 epochs while in our case we mostly trained them for 100 epochs. In our work, we wanted to get the best accuracy for the minimum amount of time and compute, so this is why we didn't explore training for more epochs. But I would expect that if you use our code base and just train the model for longer, you should get similar results as the ones reported. Let me know if you have further questions.

**Comment by gurdeepmaurya** [comment]
Thanks for clearing up my confusion.
Take Care
